test/e2e_node/numa_alignment.go

```diff
@@ -184,22 +177,35 @@ func makeEnvMap(logs string) (map[string]string, error) {
 	return envMap, nil
 }
 
-func checkNUMAAlignment(f *framework.Framework, pod *v1.Pod, logs string, numaNodes int) (numaPodResources, error) {
+type testEnvInfo struct {
+	numaNodes         int
+	sriovResourceName string
+}
+
+func containerWantsDevices(cnt *v1.Container, envInfo testEnvInfo) bool {
```

Comment:
pass the pointer on `testEnvInfo`, the same true for all appearances of `testEnvInfo`